apps/classes/models.py

```diff
@@ -1,3 +1,5 @@
+from curses.ascii import NUL
```

Comment:
what is this?

Reply:
u wrote it not me homie

Reply:
How do I write if it's in your branch LOL. It's probably from stray import. You can remove it

Reply:
IDFK